lib/config.ts

```diff
@@ -60,6 +61,8 @@ class Config {
 
     this.serviceAccountCreationTimeout = 30 * 1000; // 30 seconds
     this.serviceAccountDeletionTimeout = 30 * 1000; // 30 seconds
+
+    this.generalTimeout = 10 * 1000; // 10 seconds
```

Comment:
general timeout is currently set to 30s, why do you change it?

Reply:
It is not defined on lib/config

Reply:
I think it is in pw configuration so I just wonder why is this needed, is 30s too much for some specific action? If so, maybe you should use a different name?

Reply:
If I'm reading the documentation for timeouts correctly, for this specific use case (waitForSelector method) we do not have a default timeout set in that file. If I'm not wrong, we can't even set a default timeout for this in that file at all. 
https://playwright.dev/docs/test-timeouts

That 30-second timeout currently set in that file is for `expect()`. 

Apparently, the only way to set this specific default timeout is this here:
https://playwright.dev/docs/api/class-page/#page-set-default-timeout

But that does not change the fact, that the PW default should be indeed 30 sec... And unless you need 10 sec for some specific reason, I don't see why are you changing it.

Reply:
Ok changed to default one

Reply:
Can we resolve this conversation @Frawless @mmusil ?